src/main/java/uk/nhs/hee/tis/revalidation/connection/controller/ConnectionController.java

```diff
@@ -65,6 +65,7 @@ public class ConnectionController {
   private static final String PAGE_NUMBER = "pageNumber";
   private static final String PAGE_NUMBER_VALUE = "0";
   private static final String DESIGNATED_BODY_CODES = "dbcs";
+  private static final String TIS_DESIGNATED_BODY_CODES = "tisDesignatedBodies";
```

Comment:
So, here's a "gotcha" that got me already!  This url param is `*tis*DesignatedBodies` where elsewhere `*tcs*DesignatedBody` is used.

That's the kind of thing that increases cognitive load, debugging time and may require additional mapping customisation, as we've suffered with gmcId, gmcNumber etc.

I was on the fence and going to leave the abbreviation to `tisDbcs` alone (which also got me) but establishing & sticking to a convention should relatively simplify those concerns.

Reply:
So is that change to tcsDesignatedBodies?

I don't think tcs_ is a good naming convention, but changing the convention across the project as part of this PR would involve changes to other services

Reply:
Discussed & we can foresee the need to move to describing this as a `tis` field.  The use of the abbreviation is sufficiently okay but will need further review.